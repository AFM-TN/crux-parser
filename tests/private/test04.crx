// test array declaration

array goku : saiyan [4][3][2][1];