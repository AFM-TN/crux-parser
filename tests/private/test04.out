PROGRAM
  DECLARATION_LIST
    DECLARATION
      ARRAY_DECLARATION
        TYPE

